src/helpers.php

```diff
@@ -42,3 +42,38 @@ function backpack_avatar_url($user)
         }
     }
 }
+
+if (!function_exists('backpack_guard_name')) {
+    /*
+     * Returns the name of the guard defined
+     * by the application config
+     */
+    function backpack_guard_name()
```

Comment:
I wonder if these should be camelCase?

Reply:
Hmm... Good point. I vote no. From what I can see, [all Laravel helpers are snake_case](https://laravel.com/docs/5.6/helpers). I think we should keep that same rule, for consistency.

Reply:
*sigh* Standards only exist so they can be broken 🤦‍♂️

Reply:
No they don't. The also exist to cover all existing standards :-) https://xkcd.com/927/